glide.lock

```diff
@@ -341,7 +341,7 @@ imports:
   version: 1560c1005499d61b80f865c04d39ca7505bf7f0b
 - name: github.com/spf13/viper
   version: c975dc1b4eacf4ec7fdbf0873638de5d090ba323
-  repo: https://github.com/jimmidyson/viper.git
+  repo: https://github.com/spf13/viper.git
```

Comment:
I am also suspicious since we are referencing a commit sha here, but this sha might not be in the main repo.